roles/dnsmasq/handlers/main.yml

```diff
@@ -7,4 +14,8 @@
       networking
       {%- endif %}
     state: restarted
-  when: ansible_os_family != "CoreOS"
+  when: ansible_os_family != "RedHat"
```

Comment:
Another task could theoretically call this and break CoreOS. You should make this explicit for Debian family